visual-testing-app/src/test-utils/nested-pages.tsx

```diff
@@ -3,6 +3,7 @@ import { Route, Switch } from 'react-router-dom';
 
 type NestedPage = {
   path: string;
+  name?: string;
```

Comment:
This is just an enhancement to allow for better test rendered name so it's not tied to the URL.